.github/workflows/gen.yml

```diff
@@ -40,7 +40,8 @@ jobs:
 
       - name: Regenerate code
         shell: bash
-        run: go generate -x ./...
+        run: |
+          grep -ir "//go:generate" . | awk -F : '{ print $1 }' | uniq | xargs -n 1 -P 2 -I% go generate -x $(realpath %)
```

Comment:
This is pretty opaque, maybe a comment explaining what is going on here? Or break it up using variables?